maintenance_plan/tests/test_maintenance_plan.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import odoo.tests.common as test_common
+from odoo import fields
+from datetime import timedelta
+
+
+class TestMaintenancePlan(test_common.TransactionCase):
+
+    def setUp(self):
+        super(TestMaintenancePlan, self).setUp()
```

Comment:
Since it's Python3, you could simplify this line by writing simply:

```
super().setUp()
```

Also you can simplify the other occurrences of the super() in other files.